src/coinbase/webhook.ts

```diff
@@ -39,19 +39,23 @@ export class Webhook {
    * @param notificationUri - The URI where notifications should be sent.
    * @param eventType - The type of event for the webhook.
    * @param eventFilters - Filters applied to the events that determine which specific events trigger the webhook.
+   * @param signatureHeader - The custom header to be used for x-webhook-signature header on callbacks,
+   *   so developers can verify the requests are coming from Coinbase.
    * @returns A promise that resolves to a new instance of Webhook.
    */
   public static async create(
     networkId: string,
     notificationUri: string,
     eventType: WebhookEventType,
     eventFilters: Array<WebhookEventFilter>,
+    signatureHeader = "",
```

Comment:
The `create` args are getting large. Let's consider adding a `createWebhookOptions` type

Reply:
Example using `createTransfer` https://github.com/coinbase/coinbase-sdk-nodejs/blob/8790a49ab7ce0b55af07230b1ec5937341adb336/src/coinbase/types.ts#L862